notebooks/700_metrics/701a_aupimo.ipynb

```diff
@@ -0,0 +1,515 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    import scipy as sp
</pre><p>Can you import stats directly here?</p><p>I am looking at the import guidelines <a href="https://stackoverflow.com/a/76762035" target="_blank">https://stackoverflow.com/a/76762035</a>, <a href="https://scipy.github.io/devdocs/reference/index.html#guidelines-for-importing-functions-from-scipy" target="_blank">https://scipy.github.io/devdocs/reference/index.html#guidelines-for-importing-functions-from-scipy</a>, and this is what they recommend</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/anomalib/pull/2330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>done</p>